src/Core/Azure.DataApiBuilder.Core.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
Docs state for this setting:
> Set EmbedUntrackedSources to true to instruct the build system to embed project source files that are not tracked by the source control or imported from a source package to the generated PDB.

by " project source files that are not tracked by the source control " do they mean the actual build executable?

Reply:
I dont think they mean actual build executable but other source code files which could be ignored by source control yet be linked into the PDB, perhaps could also mean HotChocolate dependencies which are imported from a source package